spec/features/dataset_show_page_spec.rb

```diff
@@ -199,25 +199,10 @@
       expect(page).to have_content('Topic: Not added')
     end
 
-    context 'When a dataset has no datafiles' do
-      it 'Last Updated field displays last_updated_at' do
-        dataset = DatasetBuilder.new.build
-        index_and_visit(dataset)
-        expect(page).to have_content("Last updated: #{dataset['last_updated_at']}")
-      end
-    end
-
-    context 'When a dataset has datafiles' do
-      it 'Last Updated field is the most recent datafiles updated_at date' do
-        dataset = DatasetBuilder.new
-          .with_datafiles(DATA_FILES_WITH_START_AND_ENDDATE)
-          .build
-        index_and_visit(dataset)
-        datafiles = dataset[:datafiles]
-        last = datafiles.sort_by { |datafile| datafile[:updated_at] }.last
-        date = Time.parse(last["updated_at"]).strftime("%d %B %Y")
-        expect(page).to have_content("Last updated: #{date}")
-      end
+    scenario 'Last Updated field displays public_updated_at' do
```

Comment:
Is it worth testing the other paths through `displayed_date` as well?

Reply:
I'd added a couple of extra tests @rossjones 👍